frontend/src/static/js/v_zoom.js

```diff
@@ -29,9 +34,28 @@ window.vZoom = {
       filteredUser.commits = zUser.commits.filter(
           (commit) => commit[date] >= zSince && commit[date] <= zUntil,
       ).sort(this.sortingFunction);
+      this.isCommitsFinalized = true;
 
       return filteredUser;
     },
+    selectedCommits() {
+      const commits = [];
+      this.filteredUser.commits.forEach((commitDay) => {
```

Comment:
can change all the `commitDay`, `day` in code to `commit`? According to the granularity the commit here can actually be grouped by weeks etc